model_compression_toolkit/core/common/mixed_precision/mixed_precision_search_manager.py

```diff
@@ -76,6 +80,17 @@ def __init__(self,
         self.config_reconstruction_helper = ConfigReconstructionHelper(virtual_graph=self.graph,
                                                                        original_graph=self.original_graph)
 
+    @property
+    def cuts(self):
+        """
+        Calcualtes graph cuts. Written as property so it will only be calculkated once and
```

Comment:
this is the way to do cacheing? with property? (maybe consult Irena if this is the best way to do this)
also a typo